packages/webpack-scaffold/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -2,6 +2,17 @@
 
 exports[`utils Inquirer should make an Input object with validation 1`] = `
 Object {
+  "default": undefined,
```

Comment:
Don't snapshot undefined values, hard to verify if there's actually an undefined bug somewhere

Reply:
Done, I had to change the code ([link](https://github.com/webpack/webpack-cli/pull/817/commits/fb25bd2cb1d72def40e14ba431fe368cd6df6653))